app/Models/User/AccessGroupKey.php

```diff
@@ -77,7 +77,7 @@ public function filesets()
      */
     public static function getAccessGroupIdsByApiKey(string $api_key) : Collection
     {
-        return  AccessGroupKey::select('access_group_id')
+        return AccessGroupKey::select('access_group_id')
```

Comment:
can you confirm that this class is only accessed by the middleware? (Eventually, this code will be removed from DBP and added to the IAM service)

Reply:
@bradflood I removed the AccessGroupKey invocation from the AccessGroup model, as both the middleware and User/AccessGroup.php model were already accessing it. The updated code can be viewed on the following line:

https://github.com/faithcomesbyhearing/dbp/pull/917/files#diff-7cd403903b07deb6c3ba77249d41e6f9003b7aca1ee40fcc3b901c9bb3683a71L90